.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,6 +13,8 @@
   - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
   - [ ] No need for a release
 
+- [ ] **This needs to be added to LTS version!**
```

Comment:
```suggestion
- [ ] **check this box if this pull request contains fixes for LTS version**
```

Reply:
I think we should discuss more about how to notify PR authors in a better way.
 
Simply adding like this may not function as expected. Authors may not understand what is LTS.